typed-racket-test/unit-tests/static-contract-optimizer-tests.rkt

```diff
@@ -166,6 +166,9 @@
     (check-optimize (or/sc set?/sc)
       #:pos any/sc
       #:neg set?/sc)
+    (check-optimize (or/sc (or/sc set?/sc list?/sc))
+      #:pos any/sc
+      #:neg (or/sc set?/sc list?/sc))
```

Comment:
This test was just a sanity-check for me. It's not directly related to the change here.